server/src/main/java/com/linecorp/centraldogma/server/internal/api/TokenService.java

```diff
@@ -192,6 +194,43 @@ public CompletableFuture<Token> updateToken(ServiceRequestContext ctx,
         );
     }
 
+    /**
+     * PATCH /tokens/{appId}/level
+     *
+     * <p>Updates a permission of a token of the specified ID.
+     */
+    @Patch("/tokens/{appId}/level")
+    @RequiresAdministrator
+    public CompletableFuture<Token> updateTokenLevel(ServiceRequestContext ctx,
+                                                     @Param String appId,
+                                                     TokenLevelRequest tokenLevelRequest,
+                                                     Author author, User loginUser) {
+
+        checkArgument(Arrays.asList("user", "admin").contains(tokenLevelRequest.level().toLowerCase()),
+                      "Unsupported token level: " + tokenLevelRequest.level());
```

Comment:
```suggestion
        final String newTokenLevel = tokenLevelRequest.level().toLowerCase();
        checkArgument("user".equals(newTokenLevel) || "admin".equals(newTokenLevel),
                      "token level: %s (expected: user or admin)" + tokenLevelRequest.level());
```
- Let's extract `tokenLevelRequest.level().toLowerCase()` to a variable so we don't call `toLowerCase()` twice.
- Let's not create an array when calling `checkArgument`.